guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -37,7 +75,33 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 ----
 # yum install leapp-data-centos
 ----
+endif::[]
 
+ifdef::satellite[]
+.Procedure
+. Configure the repositories to obtain LEAPP.
++
+On CentOS, configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/[@theforeman/leapp COPR Repository], which contains newer LEAPP packages than those shipped by https://wiki.almalinux.org/elevate/[AlmaLinux/ELevate], and support {Project} upgrades:
++
+----
+# curl -o /etc/yum.repos.d/theforeman-leapp.repo https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/repo/epel-7/group_theforeman-leapp-epel-7.repo
+----
++
+On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
++
+----
+# subscription-manager repos --enable=rhel-7-server-extras-rpms
+----
+
+. Install required packages:
++
+----
```

Comment:
You need to specify the code block. If you check your output, you see that the attribute is not replaced.

You can use `[options="nowrap", subs="+quotes,verbatim,attributes"]`. I'd suggest you use that for every code block.